exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/HashPrelUtil.java

```diff
@@ -89,14 +91,15 @@ public LogicalExpression createCall(String funcName, List<LogicalExpression> inp
    */
   public static <T> T createHashExpression(
       List<T> inputExprs,
+      T seed,
       HashExpressionCreatorHelper<T> helper,
       boolean hashAsDouble) {
 
     assert inputExprs.size() > 0;
 
     final String functionName = hashAsDouble ? HASH32_DOUBLE_FUNCTION_NAME : HASH32_FUNCTION_NAME;
 
-    T func = helper.createCall(functionName,  ImmutableList.of(inputExprs.get(0)));
+    T func = helper.createCall(functionName,  ImmutableList.of(inputExprs.get(0), seed ));
```

Comment:
Must be, as two code lines below there is another call to createCall() with three parameters (the last of type T). Also looked -- the files (of one kind) were copied (from the other one).